File: twoSum.java
import java.util.HashMap; // import the HashMap class
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> target_diff = new HashMap<Integer, Integer>();
        int[] res=new int[2];

        for(int i=0;i<nums.length;i++){
            if(target_diff.containsKey(nums[i])){
                res[0]=target_diff.get(nums[i]);
                res[1]=i;
            }
            target_diff.put(target-nums[i],i);
        }
        return res;
    }
}

File: twoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let array = [];
    let sol_found=false;
    for(let i=0;i<nums.length && !sol_found;i++){
        tar_aux=target-nums[i];
        console.log("itera");
        for(let j=i+1;j<nums.length && !sol_found;j++){
            if(tar_aux==nums[j]){
                console.log(nums[i]);
                console.log(nums[j]);
                console.log(i);
                console.log(j);
                array.push(i);
                array.push(j); 
                sol_found=true;
            }
        }
    }
    return array;
}
console.log(twoSum([2,7,11,15],9));
